### PR TITLE
Added description to tutorials and added a tutorials.yaml

### PR DESCRIPTION
**Pull request by ahcorde**
This PR is related to this issue https://github.com/ignitionrobotics/docs/issues/55 and with these others PR: 
 - https://gitlab.com/ignitionrobotics/web/web-server/-/merge_requests/12
 - https://gitlab.com/ignitionrobotics/web/web/-/merge_requests/15

This PR includes:
 - Added `tutorial.yaml` file to get all the tutorials under the releases specified in this file.  
 - Added description for all the tutorials

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: acropolis/index.yaml [hunk 1]
```diff
@@ -18,9 +18,12 @@ pages:
   - name: install
     title: Install
     file: install.md
+    description: Citadel Installation instructions
```
Comment (nkoenig) [inline acropolis/index.yaml:21]:
```suggestion
    description: Citadel installation instructions
```

File: blueprint/index.yaml [hunk 2]
```diff
@@ -18,21 +18,28 @@ pages:
   - name: install
     title: Install
     file: install.md
+    description: Blueprint Installation instructions
```
Comment (nkoenig) [inline blueprint/index.yaml:21]:
```suggestion
    description: Blueprint installation instructions
```

File: citadel/index.yaml [hunk 3]
```diff
@@ -18,6 +18,7 @@ pages:
   - name: install
     title: Install
     file: install.md
+    description: Citadel Installation instructions
```
Comment (nkoenig) [inline citadel/index.yaml:21]:
```suggestion
    description: Citadel installation instructions
```

**Comment by maryaB-osr** [comment]
I like the set up with the cards, but I think tutorials should be under docs, just logically people will expect them there right? 
Is there a better reason to have them separate?

**Comment by chapulina** [comment]
I agree with @maryaB-osr that we should have the bigger picture in mind before deploying this so we don't have yet another page that confuses users. 

The main goal of #55 is to gather the tutorials from this repository, as well as those from all the libraries in a single index, which is not addressed by this PR.

**Pull request merged**